osquery/tables/networking/posix/curl.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <chrono>
+
+#include <boost/network/include/http/client.hpp>
+#include <boost/numeric/conversion/cast.hpp>
+
+#include <osquery/logger.h>
+#include <osquery/status.h>
+#include <osquery/tables.h>
+
+using namespace boost::network;
+using namespace boost::network::http;
+using namespace std::chrono;
+
+namespace osquery {
+namespace tables {
+
+const std::string kOsqueryUserAgent{"osquery"};
+
+Status processRequest(Row& r) {
+  r["method"] = "GET";
```

Comment:
Any reason this needs to happen here? This could also be brought down below into the `genCurl`?

Reply:
Should have ported this as well. Will change this now.